salt/modules/firewalld.py

```diff
@@ -45,11 +45,16 @@ def __firewall_cmd(cmd):
     return out['stdout']
 
 
-def __mgmt(name, _type, action):
+def __mgmt(name, _type, action, permanent=True):
     '''
     Perform zone management
     '''
-    return __firewall_cmd('--{0}-{1}={2} --permanent'.format(action, _type, name))
+    cmd = '--{0}-{1}={2} --permanent'.format(action, _type, name)
```

Comment:
I think the `--permanent` here needs to be removed. Will be added conditionally 2 lines later.